### DISCUSSION TITLE
解决的中小矿工算力机问题

### DISCUSSION THREAD

**Discussion by Chenhongwe** [filecoin-project/community-china#169] [软件优化]
**为了节约广大IPFS建设者的机器成本，我司现推出多种封装方案，价格低、效率高，无需硬件成本以及软件费用+整套封装费最低可达170Y/T,最快可1.5P/天封装效率，封装完后，对想要迁移的用户提供免费迁移解决方案，工程师上门服务。**

> `详询V：1113003652`

**Comment by dcshy** [comment]
fil出海解决方案 v：IPFS-Shy

**Reply by Chenhongwe** [reply]
呦西呦西 准备出海的可以联系我这个兄弟
